fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -33,13 +34,24 @@ def default_objective(self):
         """Return the default objective for moments of this kind."""
         return ErrorRate()
 
-    def load_data(self, X, y, event=None, **kwargs):
+    def load_data(self, X, y, event=None, multiplier=None, **kwargs):
         """Load the specified data into this object.
```

Comment:
I'm thinking we should rename `multiplier` -> `<something>_difference` since it corresponds to `g(.,1)-g(.,0)`. Some options:
* `g_difference`
* `target_difference` (since `g` is a target with respect we define the disparity)
* `utility_difference` (I like this the best, but the approach also applies if `g` is negative utility, e.g., error)
@romanlutz , @parul100495 , do you have thoughts on this?

Reply:
You mean negative utility is a weird thing to specify? I think regardless we'll need to specify that it's the difference between `g(...,1)` and `g(...,0)`. Utility seems intuitive to me as a word for this.

What I don't fully understand right now is whether this kind of formulation will prevent us from doing multi-class classification. @MiroDudik  you may have a better idea about that. Shouldn't be a blocker for this, though.

Reply:
Actually, I think that we need to change the way we do things. Instead of providing just the difference `g(.,1)-g(.,0)`, the user should provide full columns `g(.,0)` and `g(.,1)` maybe in the form of n-by-2 matrix, which could be called `g` or  `utilities` or even `costs`. This would clearly generalize well to multi-class settings.

A more important issue is that if we allow a generic `event`, I think we need to rework the derivation of the importance weights, because C^0_i and C^1_i will not line up so nicely.

Reply:
Do you think this is all in scope for this PR?

Reply:
I think so. This should only add a couple of lines in `signed_weights` and in `gamma`. I just noticed that `gamma` is currently incorrect anyway, as it is still calculating the values of constraint violation for when `g(.,0)=0` and `g(.,1)`.